apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -461,7 +461,7 @@ const trackDisableComplementaryArea = ( scope ) => {
 const trackSaveEntityRecord = ( kind, name, record ) => {
 	if ( kind === 'postType' && name === 'wp_template_part' ) {
 		const variationSlug = record.area !== 'uncategorized' ? record.area : undefined;
-		if ( document.querySelector( '.edit-site-template-part-converter__modal' ) ) {
+		if ( document.querySelector( '.edit-site-create-template-part-modal' ) ) {
```

Comment:
Hmm. Can't we extract this selector to a constant and re-use it inside React?

Reply:
Im not sure I follow, where would we be reusing it?

Reply:
Sorry for not being clear, @Addison-Stavlo.

We're [defining the UI element's class name](https://github.com/WordPress/gutenberg/blob/3053989d981309ee05694c6cde6f819891d01d84/packages/edit-site/src/components/create-template-part-modal/index.js#L45) here.

What I'm proposing is to extract this string into a variable so we could use it here as well, this way having a single source of truth. Both the linked file and the current file would use the same variable.

Reply:
Oh gotcha!

Technically that makes sense. However, its in the Gutenberg repo and I don't think there is much of an argument we can make for getting the repo to want to ensure support for exports of constants for css selectors moving forward.

Reply:
Ah, indeed, these are two different repositories. Well, never mind then! 🤷